internal/config/config.go

```diff
@@ -27,7 +27,7 @@ const (
 	DefaultEnvPrefix = "newrelic"
 
 	// DefaultLogLevel is the default log level
-	DefaultLogLevel = "INFO"
+	DefaultLogLevel = "Info"
```

Comment:
See the related comment.  We should still allow the user to use case-insensitive values.